### ISSUE TITLE
WooCommerce Subscription renewal payments reset membership start date

### ISSUE THREAD

**Issue by dwanjuki** [strangerstudios/pmpro-woocommerce#198]
**Describe the bug**
When a subscription renewal payment is made, membership startdate is reset. PMPro series progress is lost as a result.

**To Reproduce**
1. Create a subscription product that grants a membership level then buy it. 
2. Navigate to WooCommerce > Subscriptions and note the subscription ID.
2. Navigate to WooCommerce > Status > Scheduled Actions, search for the subscription ID and run the scheduled subscription payment.
3. Check the Membership Level History.
4. See that a new entry is created in level history with current day as start date.

**Expected behavior**
No change in level start date on renewal.

**Isolating the problem (mark completed items with an [x]):**
- [x] I have deactivated other plugins and confirmed this bug occurs when only Paid Memberships Pro plugin is active.
- [x] This bug happens with a default WordPress theme active, or [Memberlite](https://wordpress.org/themes/memberlite/).
- [x] I can reproduce this bug consistently using the steps above.

**WordPress Environment**
<details>
PMPro 2.12.3
PMPro WooCommerce 1.8
WooCommerce 8.2.0
WooCommerce Subscriptions 5.4.0 
WooCommerce Stripe 7.6.0
WP 6.3.2
PHP 8.0.30
</details>

**Comment by kimcoleman** [comment]
Adding additional info here - the version used to confirm this issue was Stripe - WooCommerce 7.6.0: https://wordpress.org/plugins/woocommerce-gateway-stripe/

The current version is 7.8.0.

**Comment by andrewlimaza** [comment]
@dwanjuki I've tested this again today by following the steps outlined above, can you confirm you are still facing this issue with the latest version?

My setup:

PMPro 2.12.10, PMPro Woo 1.8, Woo Stripe 8.0.0, Woo Subscriptions 5.4.0, PHP 7.4

**Comment by andrewlimaza** [comment]
I'm attaching some screenshots for reference here as we might visit this later on:

<img width="1286" alt="Screenshot 2024-03-05 at 15 47 49" src="https://github.com/strangerstudios/pmpro-woocommerce/assets/12629136/4264758c-5997-4f8c-a1b5-74c6c900db63">
<img width="1288" alt="Screenshot 2024-03-05 at 15 47 57" src="https://github.com/strangerstudios/pmpro-woocommerce/assets/12629136/79af12a6-352b-4f4a-baa7-6574a2543673">
<img width="1296" alt="Screenshot 2024-03-05 at 15 48 51" src="https://github.com/strangerstudios/pmpro-woocommerce/assets/12629136/8fc83986-965c-4020-a87b-fa90a5d96e7c">

I had cancelled my level 1 manually, then created a daily subscription product that gives me level 1 ($5 at signup and $5 recurring). I backdated the order by a day and a bit and ran the subscription scheduled action for that particular subscription and the level stayed unchanged. (Ignore the start date of the level, I was looking for row items in the Member History to confirm this was triggering).
